### DISCUSSION TITLE
Adding dev daily cards to my profile

### DISCUSSION THREAD

**Discussion by jeromehardaway** [dailydotdev/daily#907] [Feature Requests]
Recently one of my articles made it to dev.daily (https://github.com/readme/guides/engineering-career-success) and I was trying to pin it to my account. Is there a way this feature ca be added?

**Comment by idoshamun** [comment]
Hi @jeromehardaway,

I completed the process. Enjoy your awesome profile :)

**Reply by jeromehardaway** [reply]
Anyway I can add others? Also, I can start sharing links to the app? This is a fun tool.

**Reply by idoshamun** [reply]
The process should happen automatically. If it hasn't please look at step 1 here:
https://docs.daily.dev/docs/for-content-creators/claiming-ownership-on-article#how-to-claim-ownership-of-posts-you-write
Once you fix it on your side, I'll do it on our end. Just let me know
